blue_modules/notifications.ts

```
@@ -0,0 +1,629 @@
+export const getSavedUri = async (): Promise<string | null> => {
+  return await AsyncStorage.getItem(GROUNDCONTROL_BASE_URI_KEY);
+};
+
+/**
+ * Validates whether the provided GroundControl URI is valid by pinging it.
+ *
+ * @param uri - The URI to validate
+ * @returns {Promise<boolean>} TRUE if valid, FALSE otherwise
+ */
+export const isGroundControlUriValid = async (uri: string): Promise<boolean> => {
+  let response: void | Response;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)